Source/JavaScriptCore/runtime/Options.cpp

```diff
@@ -676,6 +676,16 @@ static inline void disableAllJITOptions()
     Options::needDisassemblySupport() = false;
 }
 
+#if OS(DARWIN)
+static inline void disableAllSignalHandlerBasedOptions()
+{
+    Options::usePollingTraps() = true;
+    Options::useSharedArrayBuffer() = false;
+    Options::useWasmFastMemory() = false;
+    Options::useWasmFaultSignalHandler() = false;
+}
```

Comment:
What happens when we are in WebContent process with LDM? We need to enable them.